### PR TITLE
op-guide: add clustering.

### PR DESCRIPTION
**Pull request by siddontang**
@shenli @queenypingcap

### CODE REVIEW

File: op-guide/clustering.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+                --addr 127.0.0.1:20160 \
+                --pd 127.0.0.1:2379 \
+                -S tikv 
+    ```
+
+3. Start TiDB.
+    ```bash
+    tidb-server --store=tikv \
+                --path="127.0.0.1:2379/pd?cluster=1"
+                -P 5001
```
Comment (shenli) [inline op-guide/clustering.md]: Why not use 4000? It is our official port.

**Comment by siddontang** [comment]
PTAL @shenli @iamxy

File: op-guide/docker.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    depends_on:
+      - "tikv1"
+      - "tikv2"
+      - "tikv3"
+
+    privileged: true
+```
+
++ Use `docker-compose up -d` to create and start the cluster. 
++ Use `docker-compose port tidb 4000` to print the TiDB host port. For example, if the output is `0.0.0.0:32966`, the TiDB host port is `32966`.
```
Comment (shenli) [inline op-guide/docker.md]: What's mean by "host port"?

Comment (siddontang) [inline op-guide/docker.md]:
```
docker-compose port -h
Print the public port for a port binding.
```
use public port instead? @queenypingcap

**Comment by shenli** [comment]
LGTM

**Comment by queenypingcap** [comment]
LGTM

**Pull request merged**